easybuild/easyconfigs/o/OpenMC/OpenMC-0.14.0-foss-2023a.eb

```
@@ -0,0 +1,57 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: © Copyright 2011-2023, Massachusetts Institute of Technology, 
+# 		UChicago Argonne LLC, and OpenMC contributors
+#             HPCNow!
+# Authors::   Guillermo Piccoli <guillermo.piccoli@hpcnow.com>
+# License::   MIT
+##
+
+easyblock = 'CMakeMake'
+
+name = 'OpenMC'
+version = '0.14.0'
+
+homepage = 'https://github.com/openmc-dev/openmc'
+description = """OpenMC is an open-source Monte Carlo particle transport simulation code."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+dependencies = [
+    ('HDF5', '1.14.3'),
+    ('pugixml', '1.14'),
+    ('gsl-lite', '0.41.0'),
+    ('xtensor', '0.25.0'),
+    ('fmt', '10.2.1'),
+    ('Catch2', '3.5.3')
```

```suggestion
    ('HDF5', '1.14.0'),
    ('pugixml', '1.14'),
    ('gsl-lite', '0.41.0'),
    ('xtensor', '0.25.0'),
    ('fmt', '10.1.0'),
    ('Catch2', '2.13.9'),
```
please try to use existing easyconfigs, else we get a complete dependency hell here, with ABI breakage between just about every module.